### PR TITLE
Add Database Configuration Section to Planning Doc

### PR DESCRIPTION
**Pull request by cbudz**
This PR adds a new section to the deployment planning guide that describes the CloudForms PostgreSQL database configuration. Descriptions and locations for each file are provided, along with information on user override files and querying parameters.

### CODE REVIEW

File: doc-Deployment_Planning_Guide/topics/planning.adoc [hunk 1]
```diff
@@ -259,6 +259,44 @@ Also see https://access.redhat.com/documentation/en-us/reference_architectures/2
...
+
+This section describes the {product-title} PostgreSQL database configuration. The below table provides information on each file: its location, primary function, and notes regarding behavior or recommendations. 
+
+.Database files
+[width="100%",cols="20%,20%,30%,30%",options="header",]
+|===
+|File|Location|Description|Note
+|`postgresql.conf` |Data Directory(exact location?)|Default server configuration|Adds directive to include `/etc/manageiq/postgresql.conf.d` directory
+|`postgresql.auto.conf` |Data Directory |Contains configuration set by the `ALTER SYSTEM` command|*Do not edit manually*
+|`01_miq_overrides.conf` |`/etc/manageiq/postgresql.conf.d`|Contains miq/cfme default overrides |Overwritten on upgrades
```
Comment (carbonin) [inline doc-Deployment_Planning_Guide/topics/planning.adoc]: Maybe use one of the product macros here?

File: doc-Deployment_Planning_Guide/topics/planning.adoc [hunk 2]
```diff
@@ -259,6 +259,44 @@ Also see https://access.redhat.com/documentation/en-us/reference_architectures/2
...
+|<other files>|`/etc/manageiq/postgresql.conf.d`|Contains user overrides |Takes precedence if alphabetical after `01_miq_overrides.conf`
```
Comment (carbonin) [inline doc-Deployment_Planning_Guide/topics/planning.adoc]: `s/alphabetical/alphabetically`

File: doc-Deployment_Planning_Guide/topics/planning.adoc [hunk 3]
```diff
@@ -259,6 +259,44 @@ Also see https://access.redhat.com/documentation/en-us/reference_architectures/2
...
+
+=== Database Configuration
+
+This section describes the {product-title} PostgreSQL database configuration. The below table provides information on each file: its location, primary function, and notes regarding behavior or recommendations. 
+
+.Database files
+[width="100%",cols="20%,20%,30%,30%",options="header",]
+|===
+|File|Location|Description|Note
+|`postgresql.conf` |Data Directory(exact location?)|Default server configuration|Adds directive to include `/etc/manageiq/postgresql.conf.d` directory
```
Comment (carbonin) [inline doc-Deployment_Planning_Guide/topics/planning.adoc]: This is in the `$APPLIANCE_PG_DATA` environment variable on appliances, but for this version is `/var/opt/rh/rh-postgresql95/lib/pgsql/data`

Keep in mind this *will* change in the next version of CF/ManageIQ so I'm not sure it's worth putting in here. `Data Directory` is probably good enough.

File: doc-Deployment_Planning_Guide/topics/planning.adoc (hunk 1 above)
Comment (cbudz) [reply doc-Deployment_Planning_Guide/topics/planning.adoc]: Done

File: doc-Deployment_Planning_Guide/topics/planning.adoc (hunk 3 above)
Comment (cbudz) [reply doc-Deployment_Planning_Guide/topics/planning.adoc]: Sounds good, thanks for the clarification and I agree, if its going to change, that reference is sufficient.

File: doc-Deployment_Planning_Guide/topics/planning.adoc (hunk 2 above)
Comment (cbudz) [reply doc-Deployment_Planning_Guide/topics/planning.adoc]: Thanks for catching that.

**Comment by cbudz** [comment]
@carbonin thanks for the feedback, I've implemented all suggestions and updated the PR.

**Comment by suyogsainkar** [comment]
Thanks Chris, Nick. The new section looks great - good idea to include a table calling out those .conf files. Merging to master.

**Pull request merged**